docs/storage/snapshot_restore_api.md

```diff
@@ -106,6 +106,41 @@ To wait for a restore to complete, execute:
 kubectl wait vmrestore restore-larry --for condition=Ready
 ```
 
+## Target readiness policies
+
+When restoring a `VirtualMachineSnapshot` on an already existing target (like the parent VM of the snapshot), a readiness policy can be specified to adjust how the restore happens if the target VM is not ready. The target VM is considered ready when it is fully stopped. The policy is controlled by setting `targetReadinessPolicy` to one of the available types.
+
+The following policies are available:
+- **StopTarget**: Stop the target VM so that the restore can continue immediately.
+- **WaitGracePeriod**: Wait 5 minutes for the target VM to be ready. If not ready in time, the restore will fail.
+- **FailImmediate**: Don't wait for the target to be ready before trying to restore. If it is not ready, the restore fails immediately.
+- **WaitEventually**: Kubevirt keeps the `VirtualMachineRestore` around until the target is ready. The restore is started as soon as the target is ready.
+
+## JSON patches
+
+It is sometimes necessary to modify the specs of VMs before restoring them (e.g modifying or adding annotations for CNIs).
+
+JSON patches can be applied to the `specs` of restored VMs using the `patches` parameter:
```

Comment:
As you've rightfully [mentioned](https://github.com/kubevirt/user-guide/pull/890#discussion_r2068517702), labels and annotations can also be modified. Can you please add that?